app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Models\Category;
+use Illuminate\Support\Facades\DB;
+use App\Http\Requests\AddCategoryRequest;
+use App\Http\Requests\EditCategoryRequest;
+use App\Http\Controllers\Controller;
+
+class CategoryController extends Controller
+{
+    private $controller_name = 'admin';
+    protected $path_to_view = 'admin.pages.';
+    private $path_to_ui = 'ui_resources/startbootstrap-sb-admin-2/';
+    protected $limit;
+
+    public function __construct()
+    {
+        // Var want to share
+        view()->share('controller_name', $this->controller_name);
+        view()->share('path_to_ui', $this->path_to_ui);
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    
+    public function index()
+    {
+        $categories = Category::first();
+        $this->limit = Config('app.limit');
```

Comment:
```suggestion
        $this->limit = config('app.limit');
```
config là hàm helper chúng ta viết thường nhé